sgpt/config.py

```diff
@@ -34,6 +34,9 @@
     "API_BASE_URL": os.getenv("API_BASE_URL", "default"),
     "PRETTIFY_MARKDOWN": os.getenv("PRETTIFY_MARKDOWN", "true"),
     "USE_LITELLM": os.getenv("USE_LITELLM", "false"),
+    "IN_CONTAINER": os.getenv("IN_CONTAINER", "false"),
```

Comment:
I think we should avoid using container specific ENV variables. Instead we can have only two additional vars: `OVERWRITE_SHELL_NAME` and `OVERWRITE_OS_NAME`.

Setting defaults to some specific OS/SHELL is confusing. I think we can set default values as `"default"`, later in the role definition, we can check if user wants to overwrite original OS/SHELL names.

Reply:
agreed
I find the name "default" a little confusing if the default is dynamic.
I'm opting for empty string as default value for now.

Reply:
We are using the `"default"` value for other config variables as well. Having empty string values in `.sgptrc` is confusing.